src/main/resources/openapi/usermanagement.yaml

```diff
@@ -289,16 +337,28 @@ components:
       allOf:
         - $ref: '#/components/schemas/User'
 
-    SecurityGroup:
+    SecurityGroupWithIdAndRole:
```

Comment:
Looks like id, role, global_access and display_state should also be required in the response object?

Reply:
Does `required` have meaning for response objects? I thought the intent was for `required` to apply validation constraints to incoming request objects.